src/System.Threading/src/System.Threading.csproj

```diff
@@ -16,7 +15,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Release|AnyCPU' " />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Debug|AnyCPU' " />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Release|AnyCPU' " />
-  <ItemGroup Condition="'$(TargetGroup)' == ''">
+  <ItemGroup Condition="'$(TargetGroup)' != 'net463'">
```

Comment:
I generally don't like the negative conditions but I suspect you did that because it was easier to tool. However if it is just as easy I would much prefer this be `'$(TargetGroup)` == 'netstandard1.7``.